V2

envin3 opened the pull request:
This branch will become the default one. multichain will be removed.

oliviera9 committed:
chore(husky): create dedicated commit message hook for commitlint

envin3 committed:
refactor(data-api): refactor data retrieval api functions

envin3 committed:
chore(jest): add jest support and eslint config

envin3 committed:
chore(tests): test data retrieval api functions

envin3 committed:
chore(package): remove unused packages

oliviera9 reviewed (approved):
LGTM

envin3 committed:
fix(gitHub): fix link to GitHub branch

envin3 committed:
fix(gitHub): fix link to GitHub branch

oliviera9 committed:
Merge pull request #17 from pnetwork-association/fix/link

envin3 committed:
Merge branch 'v2' into fix/link

envin3 committed:
Merge pull request #18 from pnetwork-association/fix/link

fix(gitHub): fix link to GitHub branch

envin3 committed:
feat(votes): show votes from all dao versions

envin3 committed:
Merge pull request #20 from pnetwork-association/feat/show-all-votes

feat(votes): show votes from all dao versions

envin3 committed:
chore(version): bump version and fix vulnerabilities

envin3 committed:
feat(mode): support only for native mode

envin3 committed:
Merge pull request #22 from pnetwork-association/feat/native-mode-only

Feat/native mode only

oliviera9 commented on src/components/App.jsx:
```diff
@@ -43,23 +43,15 @@ const router = createHashRouter([
 
 const { chains, provider } = configureChains(
-  [mainnet, polygon, bsc],
+  [polygon],
```
I am not completely sure it's a good idea to remove other chains from here as the provider for mainnet is used to fetch proposal for example.

envin3 replied:
This was requested.

dependabot[bot] committed:
chore(deps): bump semver and @commitlint/cli

Bumps [semver](https://github.com/npm/node-semver) to 7.5.4 and updates ancestor dependency [@commitlint/cli](https://github.com/conventional-changelog/commitlint/tree/HEAD/@commitlint/cli). These dependencies need to be updated together.


Updates `semver` from 7.3.5 to 7.5.4
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/main/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v7.3.5...v7.5.4)

Updates `@commitlint/cli` from 14.1.0 to 18.4.4
- [Release notes](https://github.com/conventional-changelog/commitlint/releases)
- [Changelog](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/cli/CHANGELOG.md)
- [Commits](https://github.com/conventional-changelog/commitlint/commits/v18.4.4/@commitlint/cli)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
- dependency-name: "@commitlint/cli"
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

oliviera9 committed:
chore(deps): bump follow-redirects

oliviera9 committed:
Merge pull request #25 from pnetwork-association/dependabot/npm_and_yarn/semver-and-commitlint/cli-7.5.4

oliviera9 commented:
Closing this PR as v2 is not the default branch
